x-pack/plugins/security/server/audit/audit_events.ts

```diff
@@ -105,7 +105,7 @@ export interface HttpRequestParams {
 }
 
 export function httpRequestEvent({ request }: HttpRequestParams): AuditEvent {
-  const { pathname, search } = request.url;
+  const { pathname, search } = request.rewrittenUrl ?? request.url;
```

Comment:
**question** should we also be preferring to use the `rewrittenUrl`'s `domain`, `port`, and `scheme` below? I feel like the answer is "yes", but I don't expect that the two URLs will differ here all that much in Production setups

Reply:
As discussed, plugins can only rewrite the path so port and scheme will always be the same